app/services/landingPageTourService.ts

```diff
@@ -0,0 +1,120 @@
+import * as angular from 'angular';
+import * as jQuery from 'jquery';
+
+interface ILandingPageTourService {
+  startTour(tourConfig: any);
+}
+
+export class LandingPageTourService implements ILandingPageTourService {
+  public static $inject = ['$document', '$timeout', 'Constants', 'GuidedTourService'];
+
+  private $document: any;
+  private $timeout: any;
+  private Constants: any;
+  private GuidedTourService: any;
+  private tourConfig: any;
+
+  constructor ($document: any, $timeout: any, Constants: any, GuidedTourService: any) {
+    this.$document = $document;
+    this.$timeout = $timeout;
+    this.Constants = Constants;
+    this.GuidedTourService = GuidedTourService;
+
+    this.tourConfig = {
+      id: "getting-started-tour",
+      steps: []
+    };
+
+    let stepConstants = Constants.LANDING_PAGE_TOUR_STEPS || {};
```

Comment:
I can completely move the tour config to the constants file to allow it to be completely over-ridden. I can define the functions that are currently used there as well. 

When we add this to console, we could put all of this into the config.js file so there is a starting point for folks to go off of. 

We could also make some defines for selectors of different components in the landing page if we feel that would be helpful.